test/Multichain.js

```diff
@@ -197,101 +221,3 @@ function makeConnectionFromEnv() {
 function makeConnectedMultichainObject() {
```

Comment:
Look into not copying this from the source; can you use the actual code? Copy paste is no bueno

Reply:
dine